### PR TITLE
Remove jQuery and update to es6

### PR DESCRIPTION
**Pull request by grafixeyehero**

### CODE REVIEW

File: Pushbullet/Web/pushbulletnotifications.html [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+                    <div class="fieldDescription">
+                        Auth key for your application
+                    </div>
+                </div>
+                <div class="inputContainer">
+                    <input is="emby-input" type="text" id="txtPushbulletDeviceId" label="PushBullet Device Id:" />
+                    <div class="fieldDescription">
+                        Device Identification
+                    </div>
+                </div>
```
Comment (h1dden-da3m0n) [inline Pushbullet/Web/pushbulletnotifications.html]: Had a quick look as to why this was missing in the 'old' config page. Seems like this is a currently completely ignored setting (except for some debug logging). However, I am not sure if we should remove it or add  it to the actual requerst sent to Pushbullet (out of scope for this PR ofc).

Ref:
* as commented in the options class: https://github.com/jellyfin/jellyfin-plugin-pushbullet/blob/fe2f27a98c21c81bdcf85c6f1f43fc3c9773ea22/Pushbullet/Configuration/PushbulletOptions.cs#L20
* and its only use so far: https://github.com/jellyfin/jellyfin-plugin-pushbullet/blob/fe2f27a98c21c81bdcf85c6f1f43fc3c9773ea22/Pushbullet/Notifier.cs#L76

**Review by h1dden-da3m0n — DISMISSED** [review]

File: Pushbullet/Web/pushbulletnotifications.html (hunk 1 above)
Comment (grafixeyehero) [reply Pushbullet/Web/pushbulletnotifications.html]: @h1dden-da3m0n i notice that maybe hide it for now or removing completely. b/c i added that the script need to get id **txtPushbulletDeviceId** value.

Comment (h1dden-da3m0n) [reply Pushbullet/Web/pushbulletnotifications.html]: I will bring it up in our matrix to see if anyone knows why it is even still there and we decide based on what I can dig up

Comment (h1dden-da3m0n) [reply Pushbullet/Web/pushbulletnotifications.html]: @grafixeyehero Okay, based on internal discussions we think its better to remove this before users get confused why the option does nothing.
If its not too much to ask could you please remove the option as a whole. (including the C# parts)

Comment (grafixeyehero) [reply Pushbullet/Web/pushbulletnotifications.html]: @h1dden-da3m0n thanks for the feed back
updated https://github.com/jellyfin/jellyfin-plugin-pushbullet/pull/28/commits/6d77f7f4764a1a1225c27abbbf2616c9d7df3f72

**Review by crobibero — DISMISSED** [review]

File: Pushbullet/Notifier.cs [hunk 2]
```diff
@@ -73,7 +73,6 @@ public async Task SendNotification(UserNotification request, CancellationToken c
             _logger.LogDebug(
                 "Pushbullet to Token : {0} - {1} - {2}",
```
Comment (h1dden-da3m0n) [inline Pushbullet/Notifier.cs]:
```suggestion
                "Pushbullet to Token : {0} - {1}",
```

**Review by h1dden-da3m0n — DISMISSED** [review]

**Review by h1dden-da3m0n — APPROVED** [review]
sorry for the logger commit

**Pull request merged**